### PR TITLE
LaTeX Style

### PR DESCRIPTION
**Pull request by bkc39**
Branch to address #5. **Do not merge this yet, it is untested**. Let me know if there are any other things that we want in these.

### CODE REVIEW

File: courseware/tex/coq311.sty [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Package Imports %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\RequirePackage[hidelinks]{hyperref}
+\RequirePackage{listings}
+\RequirePackage{amsfonts}
+\RequirePackage{amsmath}
+\RequirePackage{amssymb}
+\RequirePackage{amsthm}
+\RequirePackage{algorithm}
+\RequirePacakge{algorithmicx}
```
Comment (bennn) [inline courseware/tex/coq311.sty]: typo

File: courseware/tex/coq311.sty [hunk 2]
```diff
@@ -0,0 +1,112 @@
+%% Style file for Coq311.
+
+\ProvidesPackage{3110}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Package Imports %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\RequirePackage[hidelinks]{hyperref}
```
Comment (bennn) [inline courseware/tex/coq311.sty]: Can we change to `colorlinks`? It removes the boxes, but keeps the colors (by default: red for footnotes & magenta for urls)

File: courseware/tex/coq311.sty [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    Theorem,
+    Lemma,
+    Remark,
+    Fact,
+    Corollary,
+    Proposition,
+    Example,
+    Proof,
+    Qed,
+    Admitted
```
Comment (bennn) [inline courseware/tex/coq311.sty:87]: also, `admit`

File: courseware/tex/coq311.sty [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    Inductive,
+    CoInductive,
+    Fixpoint,
+    CoFixpoint,
+    Theorem,
+    Lemma,
+    Remark,
+    Fact,
+    Corollary,
+    Proposition,
```
Comment (bennn) [inline courseware/tex/coq311.sty:82]: I think we'll need `Program` too, but we can edit this list later. nbd.

File: courseware/tex/coq311notes.cls [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+%% Sets the picture to be shown on the title page
+\newcommand{\setnoteimage}[1]{\def\@noteimage{#1}}
+\newcommand{\show@noteimage}{\@noteimage}
+
+%% standardizes the title page
+\renewcommand\maketitle{
+  \begin{titlepage}
+    \begin{center}
+
+      \includegraphics[width=0.15\textwidth]{\show@noteimage}~\\[1cm]
```
Comment (bennn) [inline courseware/tex/coq311notes.cls:20]: Hahahah awesome. Do you have an image in mind?

File: courseware/tex/coq311pset.cls [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+%% Sets the title of the notes
+\newcommand{\setpsettitle}[1]{\def\@psettitle{#1}}
+\newcommand{\show@psettitle}{\@psettitle}
+
+%% Sets the picture to be shown on the title page
+\newcommand{\setnoteimage}[1]{\def\@noteimage{#1}}
+\newcommand{\show@noteimage}{\@noteimage}
+
+%% Sets the problem set number
+\newcommand{\setnoteimage}[1]{\def\@noteimage{#1}}
```
Comment (bennn) [inline courseware/tex/coq311pset.cls]: `setnoteimage` is already defined. Change to `setpsetnumber`, or something.

**Comment by bennn** [comment]
Every time I come to this repo it makes me really really happy.

**Comment by bkc39** [comment]
you are doing great bruh

File: courseware/tex/coq311notes.cls (hunk 5 above)
Comment (bkc39) [inline courseware/tex/coq311notes.cls:20]: either a lambda or the coq rooster.

File: courseware/tex/coq311pset.cls (hunk 6 above)
Comment (bkc39) [inline courseware/tex/coq311pset.cls]: yea that's what I had meant.

**Comment by bkc39** [comment]
All your comments above now integrated. Still untested. I plan to test drive and debug these when working on #7 .

**Comment by bkc39** [comment]
Ok, I think this one is good to merge preliminary. Give it a look and then go for it bruh.

**Comment by bkc39** [comment]
Renamed the branch `issue5` to `tex-style` so it can persist beyond this issue.

**Comment by bennn** [comment]
ok, but we can still close the pull request right?

**Comment by bkc39** [comment]
yea I'm waiting for you to close once you review.

**Pull request merged**